### PR TITLE
increase search radius for hub01-related missions

### LINKED ISSUE #66555: Hub01 Not always generating via FEMA data mission
### Describe the bug

On Latest experimental when speaking to smokes about the FEMA data mission. The messages that you are marking the location down will display as normal but it seems like 50% of the time no actual location will be added on map or in journal. 

CCDA-Fema

 

### Attach save file

N/A

### Steps to reproduce

Generate world. 
Teleport to Refugee center. 
Speak with Smokes and ask about other locations to start Unknown Recipient mission

### Expected behavior

Smokes dialogue for Unknown recipient should generate the Hub01 location or mark it's location. 

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3086 (22H2)
- Game Version: 83224ec [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    No Rail Stations [No_Rail_Stations]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by anoobindisguise**
#### Summary
none

#### Purpose of change
Fixes #66555

#### Describe the solution
Increase search radius 240 -> 2400.

#### Describe alternatives you've considered
Is there a consequence to having a gigantic search radius like this? I don't know, so if someone does please let me know and it can be reduced to like 500 or something.

### CODE REVIEW

**Comment by Procyonae** [comment]
I think peeps have said there's crashes when you set it higher than 800 iirc, a 2400 radius is pretty ridiculous I defo wouldn't merge this without in game testing

**Comment by SurFlurer** [comment]
Please test the how often hub01 will fail to generate with different search radius. To be honest, increasing the radius by 10x is too aggressive.

**Comment by RanikOrtega** [comment]
> Please test how often hub01 will fail to generate with different search radius, and justify your choice of the radius. To be honest, increasing the radius by 10x is too aggressive.

From the times I tried to create the issue report. it seems like 30-50% of the time. with default settings

File: data/json/npcs/refugee_center/surface_staff/Smokes/free_merchant_shopkeep_missions.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "om_special": "hub_01",
         "reveal_radius": 1,
         "random": true,
-        "search_range": 240
+        "search_range": 2400
```
Comment (Maleclypse) [inline data/json/npcs/refugee_center/surface_staff/Smokes/free_merchant_shopkeep_missions.json:17]:
```suggestion
        "search_range": 500
```

**Comment by Maleclypse** [comment]
This should improve it while not causing slower machines to freeze up.

**Comment by RanikOrtega** [comment]
Tried testing with 

> This should improve it while not causing slower machines to freeze up.

I tried 500 on both Smokes and Exodi bread crumb quests to Hub and 500 seems to work well.

**Pull request merged**